### DISCUSSION TITLE
ST7335s driver problem

### DISCUSSION THREAD

**Discussion by sheritafita** [arendst/Tasmota#16191] [Q&A]
Hi everyone,

I have a TFT LCD Sitronix ST7735S that I'm trying to drive by uDisplay framework.
I'm using this file : 
https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/ST7735S_M5StickC_display.ini

I have this problem : 
![297526258_1971492046377864_4518138906924681067_n](https://user-images.githubusercontent.com/78793835/183878089-4a6f0e97-0e71-4247-8e56-590c756327ee.jpg)

If someone can help please

Thanks in advance

**Comment by s-hadinger** [comment]
Did you adapt the template with the actual size of the display?

Also please paste the boot logs

**Reply by sheritafita** [reply]
Thanks @s-hadinger 

yes, I also tried another sizes but still the same :(

![Capture](https://user-images.githubusercontent.com/78793835/183881776-35958f6d-5701-406b-9ab4-448f266e81fe.PNG)

**Reply by s-hadinger** [reply]
Do you have a working code example for this display? There are many parameters in ST7735S.

**Reply by sheritafita** [reply]
unfortunately not, I only have the [datasheet](https://www.crystalfontz.com/controllers/Sitronix/ST7735S/320)
